### ISSUE TITLE
oidcAuthenticationConfiguration error

### ISSUE THREAD

**Issue by RickPoleshuck** [Call-for-Code-for-Racial-Justice/Open-Sentencing-UI#42]
It makes sense that the oidcAuthenticationConfiguration file is not committed to git, but there should be instructions on the README.md file on how to build your own or work around the problem by modifying authentication.service.ts

**Comment by amfred** [comment]
Yochana, you're working on the authentication part. right?
@yochanah

**Comment by yochanah** [comment]
Yes I am working on it.

**Comment by RickPoleshuck** [comment]
> Yes I am working on it.

@yochanah, Can I help?

**Comment by upkarlidder** [comment]
@yochanah @RickPoleshuck: so after a few hours of banging my head against the wall, I kinda have it working. I was able to create the configuration file based on the npm packages being used. I had to use IBM security verify service. Is that what you are using? Would like to compare notes next week. 

I definitely think IBM App Id is easier to configure and develop with. Is that what you are looking at?
